### PR TITLE
Add missing read-only property "outboundRules" in LoadBalancer specs

### PR DESCRIPTION
**Pull request by anton-evseev**
Copied changes for outbound rules from https://github.com/Azure/azure-rest-api-specs/pull/6493 to older API versions:
- Added read-only property "outboundRules" to backendAddressPool
- Added example for creating LB with outbound rules
- Updated other load balancer examples to reflect the fact that outbound rules are available only with Standard SKU

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by number213** [comment]
Java SDK job failed due to 2hr timeout. I guess I've updated too many API versions at once. If this is a problem, @sergey-shandar, I could split this PR in two by year (2018/2019).

**Comment by number213** [comment]
@sergey-shandar gentle ping

**Comment by number213** [comment]
@zikalino @lirenhe could you please take a look at this PR?

File: specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/loadBalancer.json [hunk 1]
```diff
@@ -1129,6 +1132,14 @@
           "$ref": "./network.json#/definitions/SubResource",
           "description": "Gets outbound rules that use this backend address pool."
         },
+        "outboundRules": {
```
Comment (yungezz) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/loadBalancer.json]: update a stable version is breaking change, pls make the change in new service tag.

**Comment by yungezz** [comment]
Hi @number213 this PR adding new property to a stable version. it's breaking change. add @MikhailTryakhov  for awareness.

**Review by yungezz — APPROVED** [review]

**Comment by yungezz** [comment]
> @yungezz I don't really see how it's a breaking change. It's a bugfix to add missing read-only property, it won't break anything on customer's side. We've already added it to 2019-06-01, this PR is to add this property to older API versions since some SDKs are multi-API

it's ok if this is a swagger file bug fixing.

**Comment by yungezz** [comment]
reopen to trigger ci.

**Pull request merged**